frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadList/uploadListItem/uploadListItem.component.tsx

```diff
@@ -76,16 +86,17 @@ export const UploadListItem = ({
 				render={({
 					field: { ref, ...extras },
 				}) => (
-					<Input
-						error={!!errors.revisionTag}
+					<UploadListItemRevisionTag
+						isSelected={isSelected}
+						errorMessage={errors.revisionTag?.message}
 						{...extras}
 					/>
 				)}
 			/>
-			<Button onClick={onClickEdit}>
+			<Button $selectedrow={isSelected} onClick={onClickEdit}>
 				<EditIcon />
 			</Button>
-			<DeleteButton onClick={onClickDelete}>
+			<DeleteButton $selectedrow={isSelected} onClick={onClickDelete}>
 				<DeleteIcon />
 			</DeleteButton>
```

Comment:
These two buttons should be the same component with no new styling `<CircleButton />` and just with different icons. 
It wont work right now because the SVGs have hardcoded colours so the need to be changed to 'currentColor'.
Also I think that CircleButton needs a little work. It is using references to fills and paths and svg. This implies that CircleButton knows the structure of the svg, if it needs to change the fill or the colour path. That's a bit unmaintainable .   It should just use css color and currentColor in the  svg icons should take care of the rest. 

For the selected colour Circlebutton can have another variant if we are going to reuse it in different lists.

Reply:
Resolved in favour of https://github.com/3drepo/3drepo.io/issues/3044